Sources/MessageKit/Controllers/MessagesViewController+Menu.swift

```diff
@@ -22,7 +22,7 @@
  SOFTWARE.
  */
 
-import Foundation
+import UIKit
```

Comment:
not sure how this compiled before, but I had to change native framework imports in a few places